scifight/admin.py

```diff
@@ -169,17 +193,38 @@ class TournamentAdmin(admin.ModelAdmin):
     def _get_short_description(self, model):
         return utils.shorten_text(model.description, maxchars=90)
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        if not request.user.is_superuser:
+            try:
+                qs = qs.filter(id=request.user.userprofile.tournament.id)
+            except:
```

Comment:
Okay, maybe that's not that pythonic way to do things, but I'd get rid of these `try`—`except` pair like the following:
```python

if not request.user.is_superuser:
    tournament_id = None
    scifight_profile = request.user.scifight_user_profile
    if scifight_profile and scifight_profile.tournament:
        tournament_id = scifight_profile.tournament.id

    qs = qs.filter(id=tournament_id)
```